bin/build-package-containing-file.js

```diff
@@ -0,0 +1,39 @@
+const fs = require( 'fs' );
+const { spawnSync } = require( 'child_process' );
+
+const args = process.argv.slice( 2 );
+if ( args.length < 1 ) {
+	console.error( 'failed to rebuild package: at least one filename must be provided' );
+	process.exit( 1 );
```

Comment:
The doc suggests

> In most situations, it is not actually necessary to call process.exit() explicitly. The Node.js process will exit on its own if there is no additional work pending in the event loop. The process.exitCode property can be set to tell the process which exit code to use when the process exits gracefully.

(https://nodejs.org/api/process.html#process_process_exitcode).

Reply:
I think that if I put this code into a function, then I could replace these with `process.exitCode = 1; return;`, but since this is all top-level code currently, this feels cleaner than `process.exitCode = 1; process.exit();`.